### DISCUSSION TITLE
Switching Users doesnt work until Browser restart.

### DISCUSSION THREAD

**Discussion by swinxx** [docmost/docmost#483] [Help / Questions] [answered]
i have tried some different user scenarios and when switching user in one browsertab, the userprofile does NOT get updated. it only works when the browsertab gets closed and reopened.. this confused me a bit. is this a common bug?

**Comment by Philipinho** [comment, marked as answer]
Fixed in [v0.6](https://github.com/docmost/docmost/releases/tag/v0.6.0). Please upgrade.
